### PR TITLE
Temporarily remove notes from the list of all services

### PR DESCRIPTION
**Pull request by dianakhuang**
to unblock provisioning.

### CODE REVIEW

**Comment by regisb** [comment]
Looks like this is failing because of `dev.check.notes`:

```
$ make dev.check."$SERVICES"
bash ./check.sh edx_notes_api
Checking edx_notes_api heartbeat:
> curl --fail -L http://localhost:18120/heartbeat
curl: (56) Recv failure: Connection reset by peer
/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/paramiko/transport.py:33: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release ofcryptography. Please upgrade your Python.
  from cryptography.hazmat.backends import default_backend
Attaching to edx.devstack.edxnotesapi
Successful checks: NONE
Failed checks: edx_notes_api_heartbeat
Check result: FAILURE
Makefile:357: recipe for target 'dev.check.edx_notes_api' failed
make: *** [dev.check.edx_notes_api] Error 2
```

Also, could you please comment out the "notes" line instead of deleting it? It will make it easier to insert it again later. A line in the README would be great as well.

**Comment by dianakhuang** [comment]
Unfortunately, I think that bash doesn't let you comment out the middle of strings. I can make the comment clearer and add something to the README, though.

File: provision.sh [hunk 1]
```diff
@@ -42,14 +42,15 @@ NC='\033[0m' # No Color
 # of services are requested.
 # Changing this order may break provisioning.
 # For example, Discovery breaks if LMS is not provisioned first.
+
+# TODO: To resolve issues with notes provisioning with ES7, we are removing it from this list until it can be added again.
```
Comment (kdmccormick) [inline provision.sh]: This list is just for establishing a provisioning order -- it doesn't actually determine what is provisioned. Instead, you want to modify `DEFAULT_SERVICES` in `options.mk`.

That will remove it from up/migrations/provisioning/etc, but still allow us to do `make dev.provision.notes` and `make dev.up.edx_notes_api`  on our own.

**Review by kdmccormick — CHANGES_REQUESTED** [review]
The idea looks good; my comment is just about an implementation detail.

Also, the **Services** section in the readme might be a good place to note this change.

File: provision.sh (hunk 1 above)
Comment (dianakhuang) [reply provision.sh]: Thanks! I was just going to let things run and see what breaks and what doesn't.

File: provision.sh [hunk 2]
```diff
@@ -42,14 +42,15 @@ NC='\033[0m' # No Color
 # of services are requested.
 # Changing this order may break provisioning.
 # For example, Discovery breaks if LMS is not provisioned first.
+
+# TODO: To resolve issues with notes provisioning with ES7, we are removing it from this list until it can be added again between forum and registrar.
```
Comment (kdmccormick) [inline provision.sh]: @dianakhuang can you revert the change to this file?

Comment (dianakhuang) [reply provision.sh]: Yup. No problem.

**Review by kdmccormick — APPROVED** [review]
LGTM! (haven't tested it myself)
